packages/shared/src/hooks/useTagAndSource.ts

```diff
@@ -101,6 +125,11 @@ export default function useTagAndSource({
 
       invalidateQueries();
 
+      invalidateContentPreferences({
+        entity: ContentPreferenceType.Keyword,
+        subQuery: RequestKey.UserFollowing,
+      });
```

Comment:
Create new function without debounce since invalidation happens for different queries then the ones shown.